components/Players/VideoPlayer.vue

```diff
@@ -4,7 +4,7 @@
       ref="videoPlayer"
       :poster="poster"
       autoplay
-      @timeupdate="onVideoProgress"
+      @timeupdate="throttledOnVideoProgress"
       @pause="onVideoPause"
       @play="onVideoProgress"
```

Comment:
```suggestion
      @play="onPlay"
```

Reply:
Is this what you had in mind? a86039b75509fd5925211c8c37719851c3375dea

Reply:
I also took the liberty to apply my shortcut to your previous fix in the audio player, tell me if it bothers you

Reply:
Done